inst/examples/shiny_interactions.R

```diff
@@ -0,0 +1,247 @@
+library(shiny)
+library(leaflet)
+library(rgdal)
+library(jsonlite)
+library(scales)
+googleLink <- function(lat,lng,zoom) {
+  sprintf("http://www.google.com/maps/place/%s,%s/@%s,%s,%sz/data=!3m1!1e3",
+          lat, lng, lat, lng, zoom)
+}
+geojson <- paste(readLines(system.file("examples/countries.json", package = "leaflet")), collapse = "\n") %>%
+  fromJSON(simplifyVector = FALSE)
```

Comment:
I believe paste and readLines are not necessary since `fromJSON()` can read from a file.